dashboard/app/controllers/featured_projects_controller.rb

```diff
@@ -0,0 +1,19 @@
+class FeaturedProjectsController < ApplicationController
+  def create
+    _, channel_id = storage_decrypt_channel_id(featured_project_params[:project_id])
+    @featured_project = FeaturedProject.create({storage_app_id: channel_id, who_featured_user_id: current_user.id})
+    @featured_project.save!
+  end
+
+  def destroy_by_project_id
```

Comment:
it seems like we should be protecting these two methods from users who do not have LEVELBUILDER or RESET_ABUSE permissions, kind of like this: https://github.com/code-dot-org/code-dot-org/blob/2caa17674cb08aed8c7e371d47c26a0b8b5e3868/dashboard/app/controllers/api/v1/pd/teacher_applications_controller.rb#L6